README.md

```diff
@@ -15,37 +15,37 @@ It simulates the diffusion and trapping of hydrogen, coupled to heat transfer wi
 
 ## Installation
 
-FESTIM can be installed via pip
+FESTIM requires FEniCS to run.
+The FEniCS project provides a prebuilt Anaconda python package (Linux and MacOS only) with FEniCS and its dependencies (python3, UFL, DOLFIN, numpy, sympy...)  already installed. Anaconda can also be used in Windows using [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) with your favourite linux distro. First [install Anaconda](https://docs.continuum.io/anaconda/install) then run the following commands:
 
-    pip install FESTIM
+    conda create -n festim-env -c conda-forge fenics
+    conda activate festim-env
 
-FESTIM requires FEniCS to run.
-The FEniCS project provides a [Docker image](https://hub.docker.com/r/fenicsproject/stable/) with FEniCS and its dependencies (python3, UFL, DOLFIN, numpy, sympy...)  already installed. See their ["FEniCS in Docker" manual](https://fenics.readthedocs.io/projects/containers/en/latest/).
+Alternativly, the FEniCS project provides a [Docker image](https://hub.docker.com/r/fenicsproject/stable/). See their ["FEniCS in Docker" manual](https://fenics.readthedocs.io/projects/containers/en/latest/) for more information.
```

Comment:
Should we simply have a link to the installation/getting started section in the documentation? Feels like we're duplicating work here

Reply:
I was wondering that myself

Reply:
Let's keep the README to a minimum and focus on the documentation.